README.md

```diff
@@ -1,2 +1,143 @@
-# hello_world
-Repo created for excercise "Set up a 'Hello Microverse' project" of Microverse program, day 2
+    <a name="readme-top"></a>
+    
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello World Excercise</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello World] <a name="about-project"></a>
+
+**[Hello World]** is a project to resolve the exercise named "Set up a 'Hello Microverse' project" of Microverse program, day 2.
+
+## 🛠 Built With HTML and CSS <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The project was developed using HTML and CSS.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Saying hello to the world.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> www.helloworld.com
+
+- [Live Demo Link](https://www.helloworld.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> This project was elaborated as an exercise. It can be used as a guideline for professional Git use.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Oluwatosin Festus Ajibola
+> Martin Ossa
+> Tshepo Makama
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/festy1)
+
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/maov19)
+
+👤 **Author3**
+
+- GitHub: [@githubhandle](https://github.com/tshepo1)
```

Comment:
- [ ] You are the only author for the repo as of now I guess, kindly remove unnecessary details. You may also want to add your `twitter` and `email` links. 😃 This will really help the visitors to your repository. 👍

Reply:
I don't have a twitter account! Maybe it's a good time to get one.
My e-mail info was added, as requested.